apps/console/src/features/applications/constants/application-management.ts

```diff
@@ -33,7 +33,9 @@ export class ApplicationManagementConstants {
     public static readonly AUTHENTICATORS_LOCAL_STORAGE_KEY: string = btoa("Authenticators");
     public static readonly EMPTY_JSON_ARRAY: string = "[]";
 
-    public static readonly SYSTEM_APPS: string[] = [ "CONSOLE","MY_ACCOUNT" ];
+    public static readonly SYSTEM_APPS: string[] = [ "CONSOLE" ];
+    public static readonly DEFAULT_APPS: string[] = [ "MY_ACCOUNT" ];
```

Comment:
Seems this variable isn't used anywhere

Reply:
I removed the `DEFAULT_APPS`, we can add it when we require it because currently my account is a default app in backend side. `SYSTEM_APPS` is a existing one. HIDDEN_APPS used for filtering.